crossfire/__init__.py

```diff
@@ -39,3 +41,17 @@ def occurrences(
         max_parallel_requests=max_parallel_requests,
         format=format,
     )
+
+
+def flatten(data, nested_columns=None):
+    if nested_columns is None:
+        nested_columns = NESTED_COLUMNS
+    nested_columns = set(nested_columns)
+    if not nested_columns.issubset(NESTED_COLUMNS):
+        raise ValueError(f"Invalid `nested_columns` value: {nested_columns}")
```

Comment:
Have you considered creating a error specific for this scenario?

Reply:
Yes, have just created a `NestedColumnError`. If you have a better sugestion for the error name, let me know..